tsconfig.spec.json

```diff
@@ -2,7 +2,7 @@
     "extends": "./tsconfig.json",
     "include": ["./src/**/__tests__/*"],
     "compilerOptions": {
-        "target": "esnext",
+        "target": "es2019",
```

Comment:
This change doesn't looks like to be relevant to the PR. I'm curious to know the reason behind this change too.

Reply:
It’s required due to the Typescript update.

Reply:
`esnext` is compatible with NodeJS 12 … but for now we can stay with it. I’ve reverted the change.